lib/src/authentication/landing_view.dart

```diff
@@ -100,7 +102,8 @@ class _LandingViewState extends State<LandingView> {
               width: double.infinity,
               child: ElevatedButton(
                 onPressed: () async {
-                  await GoRouter.of(context).replace(GlossaryView.routeName);
+                  periodicTextChange.cancel();
```

Comment:
Should be on the `dispose()`

Reply:
Updated on #262 as it should be on this PR and not this one